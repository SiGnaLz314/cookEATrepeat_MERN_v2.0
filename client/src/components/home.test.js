import React from 'react';
import renderer from 'react-test-renderer';
import Home from './home';
import { BrowserRouter } from 'react-router-dom';

const user='TestUser@testEmail.com';
const recipes=[
    {    recipe_id: 1,
        recipename: "Test",
        animal: "pork",
        ingredients: "Test",
        instructions:"Test",
        imagepath:"Test.jpg",
        imageURL: "https://cer-images.s3.amazonaws.com/images/Test.jpg",
        date: '2019-11-06T00:06:17.667+00:00',
    },
    {    recipe_id: 2,
        recipename: "Test",
        animal: "pork",
        ingredients: "Test",
        instructions:"Test",
        imagepath:"Test.jpg",
        imageURL: "https://cer-images.s3.amazonaws.com/images/Test.jpg",
        date: '2019-11-06T00:06:17.667+00:00',
    }
]; 
const admin=false;



describe('Home tests', ()=> {
    test('Home snapshot test', () => {
        const component = renderer.create(
            <BrowserRouter>
                <Home user={user}
                    recipes={recipes} 
                    admin={admin} />
            </BrowserRouter>);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
