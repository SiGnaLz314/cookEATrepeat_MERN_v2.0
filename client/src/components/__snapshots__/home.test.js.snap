// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home tests Home snapshot test 1`] = `
<div>
  <h3>
    cookEATrepeat Recipes
  </h3>
  <div
    className="container"
    id="recipe"
  >
    <div
      className="col-lg-6 box rec_img_box"
    >
      <a
        href="/recipe/1"
        onClick={[Function]}
      >
        <img
          alt="Not Available"
          id="recipe_img"
          src="https://cer-images.s3.amazonaws.com/images/Test.jpg"
        />
      </a>
    </div>
    <div
      className="col-lg-6 box rec_img_box"
    >
      <a
        href="/recipe/2"
        onClick={[Function]}
      >
        <img
          alt="Not Available"
          id="recipe_img"
          src="https://cer-images.s3.amazonaws.com/images/Test.jpg"
        />
      </a>
    </div>
  </div>
</div>
`;
